llm_studio/src/utils/config_utils.py

```diff
@@ -204,6 +204,10 @@ def load_config_yaml(path: str):
         from llm_studio.python_configs.text_causal_language_modeling_config import (
             ConfigProblemBase,
         )
+    elif cfg_dict["problem_type"] == "text_sequence_to_sequence_modeling":
```

Comment:
```
def load_config_yaml(path: str):
    """Loads config from yaml file

    Args:
        path: path of file to load from
    Returns:
        config object
    """
    with open(path, "r") as fp:
        cfg_dict = yaml.load(fp, Loader=yaml.FullLoader)

    problem_type = cfg_dict["problem_type"]
    module_name = f"llm_studio.python_configs.{problem_type}_config"
    try:
        module = importlib.import_module(module_name)
    except ModuleNotFoundError:
        raise NotImplementedError(f"Problem Type {problem_type} not implemented")
    return module.ConfigProblemBase.from_dict(cfg_dict)

```

This code snippet should work, may useful to switch to easy subsequent adding of new problem types.

Reply:
You also leave as is, as it is already part of RLHF PR